spec/support/api_spec_shared_examples.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ApiSpecSharedExamples
+  shared_context "an authenticated endpoint" do
+    response 401, "Renvoie 'unauthorized' quand l'authentification est impossible" do
+      let(:"access-token") { "false" }
+
+      schema "$ref" => "#/components/schemas/error_authentication"
+
+      run_test!
+    end
+  end
+
+  shared_context "an endpoint with access control" do |details|
+    response 403, "Renvoie 'forbidden' quand #{details}" do
+      schema "$ref" => "#/components/schemas/error_forbidden"
+
+      run_test!
+    end
+  end
+
+  shared_context "an endpoint that looks for a resource" do |details|
```

Comment:
Le nom me paraissait pas forcément parlant avant de voir ce code.

`a endpoint that returns a 404 error when a resource is not found` ? Ou c'est trop?

Reply:
J'ai fais une réécriture globale pour tâcher d'être plus explicite, sans être trop verbeux : 

"an endpoint that returns 401 - unauthorized"
"an endpoint that returns 403 - forbidden"
"an endpoint that returns 404 - not found"
"an endpoint that returns 422 - unprocessable_entity"
"an endpoint that returns 429 - too_many_requests"

Dis-moi si tu trouves ça plus clair.